src/translations.h

```diff
@@ -16,7 +16,13 @@
 #include <libintl.h>
 #include <clocale>
 
-#define _(STRING) gettext(STRING)
+extern char *blank_string;
+
+inline char *_( const char *msg )
+{
+    return ( msg[0] == '\0' ) ? blank_string : gettext( msg );
```

Comment:
Why so complicated? Just return a `const char *` and return `""` or the input string itself.

The other version that is used without `LOCALIZE` returns a `const char *` anyway.